diff --git a/.gitconfig b/.gitconfig
index 8318835..31f91db 100644
--- a/.gitconfig
+++ b/.gitconfig
@@ -1,26 +1,25 @@
 [user]
     useConfigOnly = true
-    name = Jack McKernan
-    email = jmcker@outlook.com
-    signingkey = FA8C01167D9A8CE6
+    name = (none)
+    email = (none)
 [core]
     autocrlf = input
     editor = "vim"
     excludesfile = ~/.gitignore_global
-    #hooksPath = ~/.git-templates/hooks
+    hooksPath = ~/.git-templates/hooks
     ignorecase = false
 [credential]
     username = jmcker
     helper = cache
 [commit]
-    gpgsign = true
+    gpgsign = false
 [filter "lfs"]
     required = true
     clean = git-lfs clean -- %f
     smudge = git-lfs smudge -- %f
     process = git-lfs filter-process
 [init]
-    #templateDir = ~/.git-templates
+    templateDir = ~/.git-templates
 
 # Detect Purdue
 [includeIf "gitdif:/homes/"]
