diff --git a/.gitconfig b/.gitconfig
index 05fb1b3..ddadce7 100644
--- a/.gitconfig
+++ b/.gitconfig
@@ -2,8 +2,8 @@
     useConfigOnly = true
     name = Jack McKernan
     email = jmcker@outlook.com
-    signingkey = FA8C01167D9A8CE6
-    #signingkey = B24D109150147544
+    #signingkey = FA8C01167D9A8CE6
+    signingkey = B24D109150147544
 [core]
     autocrlf = input
     editor = "vim"
@@ -11,7 +11,7 @@
     #hooksPath = ~/.git-templates/hooks
     ignorecase = false
 [credential]
-    helper = cache
+    #helper = cache
 [commit]
     gpgsign = true
 [filter "lfs"]
